cinema_project/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Django settings for cinema_project project.
+
+Generated by 'django-admin startproject' using Django 5.1.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-o9id7kd&i3*6=c&xt+56+@_#id+sfp04fh)g6pe8yr_ktmkxcl"
```

Comment:
The `SECRET_KEY` is hardcoded. For security reasons, especially in production, consider using environment variables to manage sensitive information like the secret key.